src/Razor/src/Microsoft.VisualStudio.Editor.Razor/RazorDirectiveCompletionProvider.cs

```diff
@@ -97,8 +97,13 @@ public override Task ProvideCompletionsAsync(CompletionContext context)
             }
 
             // FilePath will be null when the editor is open for cases where we don't have a file on disk (C# interactive window and others).
-            if (context.Document?.FilePath == null ||
-                !context.Document.FilePath.EndsWith(".cshtml", StringComparison.OrdinalIgnoreCase))
+            if (context.Document?.FilePath == null)
```

Comment:
Assuming we can't use content type because this is the old Roslyn API right?

Reply:
Correct